### PR TITLE
Rebuild packages/animations/example to enable macOS

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

**Comment by goderbauer** [comment]
How was this generated? Is this just deleting the `ios`, `android` etc folders and running `flutter create` again?

File: packages/animations/example/android/app/build.gradle [hunk 1]
```diff
@@ -37,12 +37,12 @@ android {
     }
 
     defaultConfig {
-        applicationId "dev.flutter.packages.animations.example"
```
Comment (goderbauer) [inline packages/animations/example/android/app/build.gradle]: Here and elsewhere: we should keep the more specific application ids.

**Comment by goderbauer** [comment]
On cirrus we currently only check that ios and android continue to build. We probably need to add checks for other platforms as well if we add them. Otherwise this can just break without us noticing...

File: packages/animations/example/android/app/build.gradle (hunk 1 above)
Comment (domesticmouse) [reply packages/animations/example/android/app/build.gradle]: Reverted this change, by re-running `flutter create --org= dev.flutter.packages.animations .`

**Comment by domesticmouse** [comment]
I know nothing about how to setup Cirrus for web and macOS. Fwiw, the web version is currently broken.

/cc @kevmoo for clues on what is required to fix all of this =)

**Comment by kevmoo** [comment]
I have no experience w/ Cirrus. Wish I did, though!

**Pull request closed without merging**